subsys/net/openthread/rpc/server/ot_rpc_message.c

```diff
@@ -118,9 +122,9 @@ static void ot_rpc_msg_udp_new(const struct nrf_rpc_group *group, struct nrf_rpc
 	ot_msg_key key;
 	struct nrf_rpc_cbor_ctx rsp_ctx;
 	otMessageSettings settings;
-	otMessageSettings *pSettings;
+	otMessageSettings *pSettings = &settings;
```

Comment:
```suggestion
	otMessageSettings *p_settings = &settings;
```
No reason in OpenThread coding stile usage in rpc

Reply:
fixed.